python/mxnet/ndarray/ndarray.py

```diff
@@ -4262,8 +4262,6 @@ def _make_dl_managed_tensor(array):
     if not ndarray.flags['C_CONTIGUOUS']:
         raise ValueError("Only c-contiguous arrays are supported for zero-copy")
     c_obj = _make_dl_managed_tensor(ndarray)
-    address = ctypes.addressof(c_obj)
-    address = ctypes.cast(address, ctypes.c_void_p)
     handle = NDArrayHandle()
-    check_call(_LIB.MXNDArrayFromDLPack(address, ctypes.byref(handle)))
+    check_call(_LIB.MXNDArrayFromDLPack(ctypes.byref(c_obj), True, ctypes.byref(handle)))
```

Comment:
why the handle here is transient handle?

Reply:
bc the DLManagedTensor here is freed immediately after this call